.github/workflows/global-ci.yml

```
@@ -292,7 +292,7 @@ jobs:
         working-directory: go-konveyor-tests
       - name: Prepare summary
         if: always()
-        run: cat /tmp/tests.log | grep -- --- >> $GITHUB_STEP_SUMMARY
+        run: [[ -f /tmp/tests.log ]] && cat /tmp/tests.log | grep -- --- >> $GITHUB_STEP_SUMMARY; true
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 295: did not find expected alphabetic or numeric character

(syntax-check)